tests/integration/targets/gem/files/tinyproxy.conf

```diff
@@ -0,0 +1,4 @@
+Port 8888
```

Comment:
Does this file allows comments? If yes, please add the copyright and license statements (as in other files) as a comment. Otherwise you need to add a `tinyproxy.conf.license` file next to it (search for `.license` files in the repo to see examples).